via/resources.py

```diff
@@ -0,0 +1,27 @@
+"""Context objects for views."""
+from pyramid.httpexceptions import HTTPBadRequest
+
+# pylint: disable=too-few-public-methods
```

Comment:
I wonder whether we should globally disable this one

Reply:
I generally think so, but I sort of enjoy being made to think about it. But 100% of the time so far I've decided to ignore it.=?